packages/back-end/package.json

```diff
@@ -20,7 +20,8 @@
     "watch-compile": "swc src -w --out-dir dist",
     "watch-dev": "delay 1 && nodemon --watch \"dist/**/*\" -e js ./dist/server.js",
     "migrate-encryption-key": "node dist/scripts/migrate-encryption-key.js",
-    "generate-api-types": "swagger-cli bundle -t yaml src/api/openapi/openapi.yaml -o generated/spec.yaml && node src/scripts/generate-openapi.mjs"
+    "generate-api-models": "node src/scripts/generate-doc-models-for-openapi-schemas.mjs",
+    "generate-api-types": "yarn generate-api-models && swagger-cli bundle -t yaml src/api/openapi/openapi.tmp.yaml -o generated/spec.yaml && node src/scripts/generate-openapi.mjs"
```

Comment:
The existing script `generate-api-types` now runs a new script `generate-api-models` before proceeding with everything. The model generation script adds all schemas programmatically to the `openapi.yaml` file, placing it in a temp file in the same directory as the existing `openapi.yaml` since the `generate-api-types` script requires it to be relatively in the same area. I had initially stored it as `/tmp/growthbook_openapi.yaml` but that broke the script, so that's why we will see a new entry in the `.gitignore`.